generic/src/test/java/org/datahem/processor/generic/GenericStreamPipelineTest.java

```diff
@@ -0,0 +1,600 @@
+import org.apache.beam.sdk.transforms.windowing.Window;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
+
+import com.google.protobuf.DescriptorProtos.FileDescriptorSet;
+import com.google.protobuf.Descriptors.FileDescriptor;
+import com.google.protobuf.DescriptorProtos.FileDescriptorProto;
+import com.google.protobuf.Descriptors.Descriptor;
+import com.google.protobuf.DynamicMessage;
+import com.google.protobuf.DynamicMessage.Builder;
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import org.datahem.processor.utils.ProtobufUtils;
+import com.google.api.services.bigquery.model.TableRow;
+import com.google.api.services.bigquery.model.TableFieldSchema;
+import com.google.api.services.bigquery.model.TableSchema;
+import com.google.api.services.bigquery.model.TableReference;
+
+import com.google.protobuf.util.JsonFormat;
+
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.DateTimeZone;
+import org.joda.time.DateTime;
+
+import org.json.JSONObject;
+import org.json.JSONArray;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import static org.junit.Assert.*;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+import java.util.Optional;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+import java.util.Collections;
+import java.util.stream.Stream;
+import java.util.Base64;
+
+import com.google.cloud.ReadChannel;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.StorageOptions;
+import com.google.common.collect.HashMultimap;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.IOException;
+import java.io.BufferedReader;
+
+import java.nio.channels.Channels;
+import java.nio.charset.StandardCharsets;
+
+@RunWith(JUnit4.class)
+public class GenericStreamPipelineTest {
+	
+	private static final Logger LOG = LoggerFactory.getLogger(GenericStreamPipelineTest.class);
+	
+	@Rule public transient TestPipeline p = TestPipeline.create();
+
+    
+	private Map<String,String> attributes = new HashMap<String, String>(){
+		{
+            put("timestamp", "2013-08-16T23:36:32.444Z");
+            put("uuid", "123-456-abc");
+            put("source", "test");
+		}
+	};
+
+
+	@Test
+	public void withoutOptionsTest(){
+
+        JSONObject testPayloadObject = new JSONObject()
+            .put("StringField", "a string")
+            .put("Int32Field", 32) 
+            .put("Int64Field", 64) 
```

Comment:
Why not a real int64 like 79239016081? So that we really make sure it's not limited to 32 bits...